catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -22,9 +22,13 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
   );
 
   return (
-    <div className={'flex flex--wrap'}>
+    <ul
+      className={'flex flex--wrap plain-list no-padding no-margin'}
+      role="list"
+      aria-labelledby="image-results"
```

Comment:
Where is this ID? I've just searched the code base and can't find it. Sorry I didn't pick up on it earlier.